src/Microsoft.ML.ImageAnalytics/ImageResizer.cs

```diff
@@ -381,7 +382,11 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                             destHeight = info.ImageHeight;
                         }
 
-                        dst = new Bitmap(info.ImageWidth, info.ImageHeight, src.PixelFormat);
+                        if ((src.PixelFormat & PixelFormat.Indexed) != 0)
+                            dst = new Bitmap(info.ImageWidth, info.ImageHeight);
```

Comment:
>new Bitmap(info.ImageWidth, info.ImageHeight); [](start = 34, length = 46)

this overload creates a bitmap with a PixelFormat enumeration value of Format32bppArgb. So we're dropping the src's indexed format and forcing it to be Format32bppArgb? Is that ok?

[https://docs.microsoft.com/en-us/dotnet/api/system.drawing.bitmap.-ctor?view=netframework-4.8#System_Drawing_Bitmap__ctor_System_Int32_System_Int32](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.bitmap.-ctor?view=netframework-4.8#System_Drawing_Bitmap__ctor_System_Int32_System_Int32)_

Reply:
32bppArgb is ok. @shmoradims read more here: https://stackoverflow.com/questions/17313285/graphics-on-indexed-image

Reply:
Maybe it would be good to stick a comment in here on why this is necessary... Up to you.

---
In reply to: [279062683](https://github.com/dotnet/machinelearning/pull/3601#discussion_r279062683) [](ancestors = 279062683)

Reply:
Please see my response below.

Reply:
Done.

---
In reply to: [279069616](https://github.com/dotnet/machinelearning/pull/3601#discussion_r279069616) [](ancestors = 279069616,279062683)